src/cases/use-cases/change-case-stage.use-case.js

```diff
@@ -24,6 +24,21 @@ export const changeCaseStageUseCase = async (caseId) => {
     );
   }
 
+  const allTasksStatus = [];
```

Comment:
Maybe outside the scope of a bug ticket but most of this could be refactored DDD style and pushed down into the case domain, something like:

```
const kase = await findById(caseId);

kase.nextStage();

return update(kase);
```